### PR TITLE
Get Started Documention

### LINKED ISSUE #3: Document how to get started

### PR DESCRIPTION
**Pull request by CodeStrate**
A non-code contribution that is to close #3 

Added:
- [x] Reddit API Instructions. (with images,)
- [x] Installation Prerequisites.
- [x] Dependencies.
- [x] Local Bot Usage.
- [x] Contributing Section (open to be amended by [Contributing Readme](#CONTRIBUTING.md))

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+- [Contributing](#contributing)
+
+# Getting Started
+
+Follow these steps to set up and run the Factorion bot on your local machine and account.
+
+## Prerequisites
+- **Rust** (latest stable version) - [Install Rust](https://www.rust-lang.org/tools/install)
+- **Cargo** - This comes installed with Rust.
+- **Reddit Account** - To create a bot, you'll need a Reddit account. [Reddit](https://www.reddit.com/)
```
Comment (tolik518) [inline README.md:31]:
```suggestion
## Prerequisites
- **Rust** (latest stable version) - [Install Rust](https://www.rust-lang.org/tools/install)
- **Reddit Account** - To run the bot, you'll need a Reddit account. [Reddit](https://www.reddit.com/)
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  
+## Reddit API Credentials
+###### You can checkout [Reddit API Documentation](https://www.reddit.com/dev/api) to checkout all the different endpoints you can access. 
+1. We need `Application ID` and `Secret Key` so that Reddit can know about our app. [preferences/apps](https://www.reddit.com/prefs/apps)
+2. Click the <b>are you a Developer?</b> button to start setting up the bot.
+
+![image](https://github.com/user-attachments/assets/140056ac-91ce-4178-8703-19451357adce)
```
Comment (tolik518) [inline README.md:38]:
```suggestion
<details>
<summary>Screenshot</summary>
![image](https://github.com/user-attachments/assets/140056ac-91ce-4178-8703-19451357adce)
</details>
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+3. Fill in the required details:
+   - **Name**: Choose a name for your bot.
+   - **App type**: Select **Script**.
+   - **Redirect URI**: Use `http://localhost:8080` (or any URI; it’s not used for script bots).
+     
+![image](https://github.com/user-attachments/assets/2450994a-14cf-4f46-9f71-518ceb0c59f5)
```
Comment (tolik518) [inline README.md:45]:
```suggestion
<details>
<summary>Screenshot</summary>
![image](https://github.com/user-attachments/assets/2450994a-14cf-4f46-9f71-518ceb0c59f5)
</details>
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+```env
+CLIENT_ID=<your_client_id>
+CLIENT_SECRET=<your_client_secret>
+USERNAME=<reddit_app_username>
+PASSWORD=<reddit_app_password>
+```
+
+Replace `<your_client_id>`, `<your_client_secret>`, `<reddit_app_username>`, and `<reddit_app_password>` with the values you received from the Reddit App creation.
+
+### Dependencies
```
Comment (tolik518) [inline README.md:75]: Dependencies may change and are easy accessible when looking at the Cargo.toml, so I think you can remove that section :)

File: README.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+### Configuration
+
+Create a `.env` file in the project root with the following variables:
+
+```env
+CLIENT_ID=<your_client_id>
+CLIENT_SECRET=<your_client_secret>
+USERNAME=<reddit_app_username>
+PASSWORD=<reddit_app_password>
```
Comment (tolik518) [inline README.md:70]:
```suggestion
CLIENT_ID=<your_client_id>
CLIENT_SECRET=<your_client_secret>
USERNAME=<reddit_app_username>
PASSWORD=<reddit_app_password>
SUBREDDITS=<subreddits>
```

**Comment by tolik518** [comment]
Thanks for the PR!
I've added some comments and suggestions. If they're resolved, I can merge

**Comment by CodeStrate** [comment]
@tolik518 Done the changes can you check?

**Comment by tolik518** [comment]
Looks good to me, thank you :)

**Pull request merged**